pintc/src/error/handler.rs

```diff
@@ -25,19 +25,44 @@ impl Handler {
         ErrorEmitted { _priv: () }
     }
 
+    /// Emit the warning `warn`.
+    pub fn emit_warn(&self, warning: Warning) -> WarningEmitted {
+        self.inner.borrow_mut().warnings.push(warning);
+        WarningEmitted { _priv: () }
+    }
+
```

Comment:
I doubt `WarningEmitted` is going to be useful.. I suggest we remove it for now.